app/code/Magento/MediaContentApi/Model/GetAssetIdByContentFieldInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContentApi\Model;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * Interface used to return Asset id by content field.
+ */
+interface GetAssetIdByContentFieldInterface
```

Comment:
Let's use plural to indicate that return type is an array
```suggestion
interface GetAssetIdsByContentFieldInterface
```